onebot/plugins/karma.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+================================================
+:mod:`onebot.plugins.karma` Karma
+================================================
+
+This plugin keeps track of karma.
+
+To modify: foo++ / (foo bar)++ / foo--
+To view: !karma foo bar
+"""
+
+import base64
+import irc3
+from irc3 import plugin
+from irc3.plugins.command import command
+
+
+@plugin
+class KarmaPlugin(object):
+    """Karma Plugin"""
+
+    requires = [
+        'irc3.plugins.storage',
+        'irc3.plugins.command',
+    ]
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @staticmethod
+    def _find_key(data, modifier='++'):
```

Comment:
Hmm, this is new to me. I've updated the code. Do you have a reference? I'm having trouble finding anything about this.